packages/misc/angular-in-memory-web-api/package.json

```
@@ -5,10 +5,12 @@
   "author": "angular",
   "license": "MIT",
   "peerDependencies": {
-    "@angular/core": "^8.0.0",
-    "@angular/common": "^8.0.0",
-    "rxjs": "^6.5.3",
-    "tslib": "^1.10.0"
+    "@angular/core": "^12.0.0-next.5",
+    "@angular/common": "^12.0.0-next.5",
```

OOC, why these changes?

---

Because older versions will no longer be supported by this change. Also, the peer dep was wrong in the first place

---

But then shouldn't these be:

```suggestion
    "@angular/core": "^12.0.0-next.6",
    "@angular/common": "^12.0.0-next.6",
```